src/main/resources/application.properties

```diff
@@ -29,6 +29,6 @@ quarkus.log.handler.console."STRUCTURED_LOGGING".format=date="%d{yyyy/MM/dd}" ti
 quarkus.log.handler.console."STRUCTURED_LOGGING".enable=true
 
 # service logging config
-quarkus.log.category."com.wire.bots".use-parent-handlers=false
-quarkus.log.category."com.wire.bots".level=INFO
-quarkus.log.category."com.wire.bots".handlers=STRUCTURED_LOGGING
\ No newline at end of file
+quarkus.log.category."com.wire.integrations.jvm".use-parent-handlers=false
+quarkus.log.category."com.wire.integrations.jvm".level=INFO
+quarkus.log.category."com.wire.integrations.jvm".handlers=STRUCTURED_LOGGING
```

Comment:
Question: This config will apply to the SDK. Do we need a dedicated one for the reminder app ? 🤔